src/sentry/api/event_search.py

```diff
@@ -194,6 +197,29 @@ def translate_wildcard(pat: str) -> str:
     return "^" + res + "$"
 
 
+def translate_escape_sequences(string: str) -> str:
```

Comment:
Could we add unit tests for this function directly?

Reply:
added in 0f0c5ff